client/packages/invoices/src/Prescriptions/DetailView/columns.ts

```diff
@@ -253,29 +237,25 @@ export const usePrescriptionColumn = ({
           },
         },
       ],
-      [
-        'packSize',
-        {
-          getSortValue: row => {
-            if ('lines' in row) {
-              const { lines } = row;
-              return (
-                ArrayUtils.ifTheSameElseDefault(lines, 'packSize', '') ?? ''
-              );
-            } else {
-              return row.packSize ?? '';
-            }
+      {
+        key: 'packUnit',
+        label: 'label.pack',
+        sortable: false,
+        Cell: PackUnitMultipleCell({
+          getItemId: row => {
+            if ('lines' in row) return '';
```

Comment:
Should probably refactor prescriptions since it's not being grouped...

Reply:
idk if we wanna group it that's why I've kept it in... but maybe just removing all the grouped code might be better for prescriptions